static/src/javascripts/projects/common/modules/commercial/contributions-service.js

```diff
@@ -6,12 +6,13 @@ import {
 import { mountDynamic } from '@guardian/automat-modules';
 import { onConsentChange } from '@guardian/consent-management-platform';
 import userPrefs from 'common/modules/user-prefs';
-import config from '../../../../lib/config';
-import { getCookie } from '../../../../lib/cookies';
-import fastdom from '../../../../lib/fastdom-promise';
-import { fetchJson } from '../../../../lib/fetch-json';
-import { getCountryCode } from '../../../../lib/geolocation';
-import reportError from '../../../../lib/report-error';
+import { storage } from '@guardian/libs';
+import config from 'lib/config';
+import { getCookie } from 'lib/cookies';
+import fastdom from 'lib/fastdom-promise';
+import { fetchJson } from 'lib/fetch-json';
+import { getCountryCode } from 'lib/geolocation';
+import reportError from 'lib/report-error';
```

Comment:
Any reason for not using the relative paths?

https://github.com/guardian/frontend/pull/23494

Reply:
oh right, I thought it looked nicer this way! I'll change

Reply:
Intellij is sad
![Screen Shot 2021-08-13 at 09 03 22](https://user-images.githubusercontent.com/1513454/129324966-a3922d33-6064-4008-b7ce-0b28d3534ece.png)

Reply:
I agree, it’s generally nicer, we’ll only ask for files that end up in the commercial bundle.